readme.md

```diff
@@ -54,6 +54,9 @@ All security vulnerabilities should be reported in accordance with our
 Directus is a premium open-source ([GPLv3](./license)) project, made possible with support from our passionate core
 team, talented contributors, and amazing [GitHub Sponsors](https://github.com/sponsors/directus). Thank you all!
 
+See [Running Locally](https://docs.directus.io/contributing/running-locally.html) to get a local development version up
+and running.
+
```

Comment:
Let's omit that in favor of https://github.com/directus/directus/pull/18133#issuecomment-1503846830